cypher/cypher-docs/src/docs/graphgists/query-tuning/advanced-query-tuning-example.asciidoc

```diff
@@ -1,4 +1,23 @@
-= Advanced query tuning example
+[[cypher-index-values-order]]
+= Index Values and Order
+
+[abstract]
+--
+This section describes some more subtle optimizations based on new native index capabilities
+--
+
+One of the most important and useful ways of optimizing Cypher queries involves creating appropriate indexes.
+This is described in more detail in the section on '<<query-schema-index,Indexes>>', and demonstrated in the <<cypherdoc-basic-query-tuning-example,basic query tuning example>>.
+In summary an index will be based on the combination of a `Label` and a `property`.
+Any Cypher query that searches for nodes with a specific label and some predicate on the property (equality, range or existence) will be planned to use
+the index if the cost planner deems that to be the most efficient solution.
+
+In order to benefit from some recent enhancements added to Neo4j 3.5, it is useful to understand when _index-backed property lookup_ and _index-backed order-by_ will come into play.
```

Comment:
I don't know if it is the case already, but I think a link to this section from the "ORDER BY" clause page would be great. In this way, if people read that, they will know that Neo now supports index-backed ordering....

Reply:
I suggest - for all cases (and there are a few) - that "index-backed order-by" be changed to "index-backed order by"